python/rikai/spark/sql/codegen/sklearn.py

```diff
@@ -0,0 +1,46 @@
+#  Copyright 2021 Rikai Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Iterator
+import pandas as pd
+import numpy as np
+
+from pyspark.sql.types import StructType
+from pyspark.sql.functions import pandas_udf
+
+
+def generate_udf(spec: "rikai.spark.sql.codegen.base.ModelSpec"):
```

Comment:
Can you add a unit test for this? LogisticRegression or RandomForest would probably be the most commonly encountered